src/js/profile/mobile/widget/Calendar.js

```diff
@@ -0,0 +1,275 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable block-scoped-var */
+/* eslint-disable no-eq-null */
+/* eslint-disable one-var */
+/* eslint-disable vars-on-top */
+/*global window, define, ns */
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * @example
+ * <div class="ui-calendar"></div>
+ *
+ * @since 1.2
+ * @class ns.widget.core.Calendar
+ * @extends ns.widget.BaseWidget
+ * @author Dohyung Lim <delight.lim@samsung.com>
+ */
+(function (document, ns) {
+	"use strict";
+	//>>excludeStart("tauBuildExclude", pragmas.tauBuildExclude);
+	define(
+		[
+			"../../../core/engine",
+			"../../../core/util/DOM/attributes",
+			"../widget"
+		],
+		function () {
+			//>>excludeEnd("tauBuildExclude");
+			var utilsObject = ns.util.object,
+				engine = ns.engine,
+				events = ns.event,
```

Comment:
For using the "ns.event" , 
shouldn't you add    "../../../core/event" to the define?
It means
	define(
		[
			"../../../core/engine",
			"../../../core/util/DOM/attributes",
                         "../../../core/event",
			"../widget"
		],

Reply:
If "../../../core/event" is added, all the code of the widget is fetched, but ns.event fetches only the event content.
No modification is required, because already get 'ns' as a parameter in 31 line.

Reply:
I agree the @singa2000 mention. It would be good to add the event.
If there is no widget to add the 'core/event', it make the error.

Reply:
I think that an error will not occur because it is a widget that essentially uses an event. 
Is it better to add it though?

Reply:
If we only import the calendar widget only, I think it can occurr the error.

Reply:
I think it is better to fix if there is room for error. I'll edit it.